silo-core/deploy/silo/SiloDeploy.s.sol

```diff
@@ -283,6 +299,17 @@ contract SiloDeploy is CommonDeploy {
         isDiaOracle = diaOracle != address(0);
     }
 
+    function _isFixedAddress(string memory _oracleConfigName) internal pure returns (address fixedAddress) {
```

Comment:
Seems like this can be replaced by `Strings.parseAddress()` from OZ.

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/ad906fe39b643dd1bfbadc8480a631d223a75d6a/contracts/utils/Strings.sol#L362C14-L362C26

Reply:
I reuse some code, but I can not use it doirecly form OZ, because in version we have, this code does not exists.